make/data/cldr/common/properties/coverageLevels.txt

```diff
@@ -1,141 +1,143 @@
-yue_Hans ;	modern
-zh ;	modern
-zh_Hant ;	modern
-zu ;	modern
-#EOF
+# coverageLevels.txt		
+# Copyright © 2023 Unicode, Inc.		
+# CLDR data files are interpreted according to the		
+# LDML specification: http://unicode.org/reports/tr35/		
+# For terms of use, see http://www.unicode.org/copyright.html		
+#		
+# For format and usage information, see:		
+# https://cldr.unicode.org/index/cldr-spec/coverage-levels.		
+		
+		
+af ;	modern ;	Afrikaans
+am ;	modern ;	Amharic
+ar ;	modern ;	Arabic
+as ;	modern ;	Assamese
+ast ;	basic ;	Asturian
+az ;	modern ;	Azerbaijani
+be ;	modern ;	Belarusian
+bg ;	modern ;	Bulgarian
+bgc ;	basic ;	Haryanvi
+bho ;	basic ;	Bhojpuri
+bn ;	modern ;	Bangla
+br ;	moderate ;	Breton
+brx ;	basic ;	Bodo
+bs ;	modern ;	Bosnian
+bs_Cyrl ;	basic ;	Bosnian (Cyrillic)
+ca ;	modern ;	Catalan
+ceb ;	moderate ;	Cebuano
+chr ;	moderate ;	Cherokee
+cs ;	modern ;	Czech
+cv ;	moderate ;	Chuvash
+cy ;	modern ;	Welsh
+da ;	modern ;	Danish
+de ;	modern ;	German
+doi ;	basic ;	Dogri
+dsb ;	moderate ;	Lower Sorbian
+el ;	modern ;	Greek
+en ;	modern ;	English
+es ;	modern ;	Spanish
+et ;	modern ;	Estonian
+eu ;	modern ;	Basque
+fa ;	modern ;	Persian
+ff_Adlm ;	basic ;	Fula (Adlam)
+fi ;	modern ;	Finnish
+fil ;	modern ;	Filipino
+fo ;	moderate ;	Faroese
+fr ;	modern ;	French
+ga ;	modern ;	Irish
+gd ;	modern ;	Scottish Gaelic
+gl ;	modern ;	Galician
+gu ;	modern ;	Gujarati
+ha ;	modern ;	Hausa
+he ;	modern ;	Hebrew
+hi ;	modern ;	Hindi
+hi_Latn ;	modern ;	Hindi [Latin]
+hr ;	modern ;	Croatian
+hsb ;	moderate ;	Upper Sorbian
+hu ;	modern ;	Hungarian
+hy ;	modern ;	Armenian
+ia ;	basic ;	Interlingua
+id ;	modern ;	Indonesian
+ig ;	modern ;	Igbo
+is ;	modern ;	Icelandic
+it ;	modern ;	Italian
+ja ;	modern ;	Japanese
+jv ;	modern ;	Javanese
+ka ;	modern ;	Georgian
+kea ;	basic ;	Kabuverdianu
+kgp ;	basic ;	Kaingang
+kk ;	modern ;	Kazakh
+km ;	modern ;	Khmer
+kn ;	modern ;	Kannada
+ko ;	modern ;	Korean
+kok ;	modern ;	Konkani
+ks ;	basic ;	Kashmiri
+ks_Deva ;	basic ;	Kashmiri (Devanagari)
+ky ;	modern ;	Kyrgyz
+lo ;	modern ;	Lao
+lt ;	modern ;	Lithuanian
+lv ;	modern ;	Latvian
+mai ;	basic ;	Maithili
+mi ;	basic ;	Māori
+mk ;	modern ;	Macedonian
+ml ;	modern ;	Malayalam
+mn ;	modern ;	Mongolian
+mni ;	basic ;	Manipuri
+mr ;	modern ;	Marathi
+ms ;	modern ;	Malay
+my ;	modern ;	Burmese
+ne ;	modern ;	Nepali
+nl ;	modern ;	Dutch
+nn ;	modern ;	Norwegian Nynorsk
+no ;	modern ;	Norwegian
+or ;	modern ;	Odia
+pa ;	modern ;	Punjabi
+pcm ;	modern ;	Nigerian Pidgin
+pl ;	modern ;	Polish
+ps ;	modern ;	Pashto
+pt ;	modern ;	Portuguese
+qu ;	moderate ;	Quechua
+raj ;	basic ;	Rajasthani
+rm ;	basic ;	Romansh
+ro ;	modern ;	Romanian
+ru ;	modern ;	Russian
+sa ;	basic ;	Sanskrit
+sat ;	basic ;	Santali
+sc ;	moderate ;	Sardinian
+sd ;	modern ;	Sindhi
+sd_Deva ;	basic ;	Sindhi (Devanagari)
+si ;	modern ;	Sinhala
+sk ;	modern ;	Slovak
+sl ;	modern ;	Slovenian
+so ;	modern ;	Somali
+sq ;	modern ;	Albanian
+sr ;	modern ;	Serbian
+sr_Latn ;	modern ;	Serbian (Latin)
```

Comment:
@naotoj BTW this was fixed

Reply:
Yes. The patch before was removed with this commit: https://github.com/openjdk/jdk/pull/13469/commits/0e3456c8924960dc0bc488b08b9763bd08557a68